app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,22 @@
 <!DOCTYPE html>
+<html lang="de">
 <html>
   <head>
-    <title>BelegarbeitRor</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+    <title>Belegarbeit</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
```

Comment:
Das ist falsch. So haben wir es nicht in der Vorlesung gemacht und so ist es auch nicht in Gem Bootstrap beschriebe. Schaut bitte dazu die Doku zum Gems an.
https://github.com/twbs/bootstrap-rubygem